wasm/src/glob.cc

```diff
@@ -101,6 +103,16 @@ opa_value *opa_glob_match(opa_value *pattern, opa_value *delimiters, opa_value *
             return NULL;
         }
 
+		if (c->size() >= MAX_CACHE_SIZE)
```

Comment:
I still need to add tests for this and the regex built-in.

Reply:
Resolved, isn't it?